### PR TITLE
Hello Microverse project pull request

### PR DESCRIPTION
**Pull request by Mukaba**
## **Creating a Styled Webpage with Linter Error Checking** 

This project involved creating an `index.html` and `style.css` file in a feature branch, and adding a styled `h1` heading. Additionally, the project was linked to a Linter for error checking which passed all tests. The following points highlight the changes made in this milestone: 

- Created the `index.html` and `style.css` files in the feature branch (according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules). 
- Added a styled `h1` heading in the `style.css` file. 
- Linked the project to Linter for error checking. 
- Ran tests with the Linter to ensure that no errors were present. 
- Used Markdown syntax to make the project description more appealing.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (adel-gu) [inline index.html:1]: - [x] Please consider adding a descriptive title and a summary for this PR explaining point by point your achievements and the changes made in this milestone🎯 you can use the `markdown` syntax to make the PR description more appealing, for more information on how you can write an [MarkDown](https://www.markdownguide.org/basic-syntax/).

image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
```
Comment (adel-gu) [inline style.css:1]: - [ ] Please use the [Microverse template](https://github.com/microverseinc/readme-template) to document your repo, I notice that the main branch already has it, so merging this branch will cause changing the whole REAMDE content in the main branch, so Kindly suggest adding The Microverse  README to this branch as well.

**Review by adel-gu — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES ♻

**Hi @Mukaba 👋🏼,** 

Good job so far! 👏👏👏 you've accomplished the majority of the project requirement really well 😎, However, There are some issues that you still need to work on to go to the next project but you are almost there! 💪


## Highlights
- Linter is set up for this project ✔️ 
- All linter checks are passed  ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mukaba** [comment]
## **Creating a Styled Webpage with Linter Error Checking** 

This project involved creating an `index.html` and `style.css` file in a feature branch, and adding a styled `h1` heading. Additionally, the project was linked to a Linter for error checking which passed all tests. The following points highlight the changes made in this milestone: 

- Created the `index.html` and `style.css` files in the feature branch (according to the [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) rules). 
- Added a styled `h1` heading in the `style.css` file. 
- Linked the project to Linter for error checking. 
- Ran tests with the Linter to ensure that no errors were present. 
- Used Markdown syntax to make the project description more appealing.

**Review by Gambit142 — APPROVED** [review]
Hi @Mukaba 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- `node_modules` and the `test.md` file are git ignored ✔️ 

### Optional Suggestion

- [ ] Currently, your _**README.md**_ file is not so professional, to make it more professional, kindly do the following:

- Include 1-3 features of this project in the **_Key Features_** section
- Remove the **_Live Demo_** Section
- In the Getting started section, do the following:
  - For the _**Prerequisite**_ section, include packages like Git, Npm, Node and VS Code that one needs to install before cloning this project
  - For the _**Setup**_ section, give information that shows how to clone the repository
  - For the _**Install**_ section, show the ways to install linters you used for the project
  - You can remove the _**Usage**_, the _**Deployment**_ and **_Run Test_** subsections because they won't be needed for this project
- Include 1-3 features that can be implemented in the **_Future Features_** section. Features like building a webpage, navigation or whatever you desire
- The **_Show Your Support_** Section should contain information about how the outside world can support the project. Could you indicate none for now or remove it?
- The _**Acknowledgement**_ section should appreciate those who contributed to the success of the project


#### It is important you get used to making your REDAME professional because it is a general requirement for all projects in Microverse. It also builds and improves your documentation skills which are important in our field of study. 👍 💻 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**